cmd/notification.go

```diff
@@ -672,6 +673,24 @@ func (sys *NotificationSys) SetBucketSSEConfig(ctx context.Context, bucketName s
 	}()
 }
 
+// SetBucketVersioning - calls SetBucketVersioning on all peers.
+func (sys *NotificationSys) SetBucketVersioning(ctx context.Context, bucketName string,
+	v *versioning.Versioning) {
+	go func() {
```

Comment:
It is just a notification @klauspost if they fail, they reload from disk on a timely basis @klauspost

Reply:
Isn't this setting the state on the remote servers though, or am I completely mistaking?

Is seems to me that it would be possible to end up in a desync'ed state where some servers think a bucket has versioning while others don't?

I may be completely off, so feel free to correct me.

Reply:
Yes, that is indeed possible but it's not easily fixable or made atomic @klauspost because rollbacks are not truly possible as the rollbacks itself may fail due to network issues.

Reply:
This will be resolved with #9586